### ISSUE TITLE
Update in Linux 

### ISSUE THREAD

**Issue by andregarban** [zim-desktop-wiki/zim-desktop-wiki#2170]
After updating in Linux, it kept my structure but no texts are appearing. I sync my notes with Dropbox and they are still there, but not showing on my original structure!

**Comment by patitude29** [comment]
Similar issue. I'm on linux using zim version 0.75.0 and my zim files sync to dropbox. After yesterday's update, all my files are intact but none of the notebooks are opening. Instead, I'm getting one notebook from 2015 with all of my other notebooks listed, but none of the files are available except the ones that go to the 2015 notebook.

Update: I rolled back to version 0.72.0 and the notebooks open correctly.

**Comment by Se7enLC** [comment]
I just experienced the same thing. After updating and re-launching zim, it popped up a dialog that said it was building an index. My notes were all gone -- moved into a folder "notes" under the top-level. This broke the Journal feature, among other things.

I was able to fix it after some debugging. Under File | Open Another Notebook, I saw my expected notebook listed. At the bottom there is a selection labeled "Default Notebook" which had nothing selected. I selected my notebook, hit ok, and then I closed zim.

Next, i found and deleted the notebook.zim file that had been created in ~/zim for some reason. When I re-opened zim, it now properly opened my notebook.

**Comment by andregarban** [comment]
> Similar issue. I'm on linux using zim version 0.75.0 and my zim files sync to dropbox. After yesterday's update, all my files are intact but none of the notebooks are opening. Instead, I'm getting one notebook from 2015 with all of my other notebooks listed, but none of the files are available except the ones that go to the 2015 notebook.
> 
> Update: I rolled back to version 0.72.0 and the notebooks open correctly.

Yes, that worked for me as well! No update by now. Hopefully the bug will be fixed-

**Comment by jaap-karssenberg** [comment]
Afraid I need a little bit more detailed description of what it is that you see exactly. Can you elaborate on what you see when you say the notes were "moved" ?

Issues with opening notebooks in the first place might be related to #2165

**Comment by jaap-karssenberg** [comment]
Also, please use a more descriptive title for your issue next time. The more structured you fill in the information the easier it is for us to actually fix the issue. Please use the issue templates for this.

**Comment by SenorFusion** [comment]
Something similar happened to me. 

I just updated to 0.75. 
Fedora 36

This is what I saw

1. Open Zim as normal, from quick launch menu
2. See "Choose Notebook" diaglog (This is not normal, I only have 1 notebook, it always just opens)
3. Simultaneous to 2, see "Rebuilding Index" dialog
4. Notebook opens but it is not the same as before
5. in TOC sidepane there is now a new Top Level File/Folder called _<name_of_notebook>_  with creation date <today> and all other folders are nested under this (so, one level lower than they were before). The page itself is empty
6.  "home" page variable set in notebook.zim is not respected - defaults to this new top-level page
7. If I press the "Home" button (top left) It takes me to an empty page titled "Home", creation date <today> even though I have a "Home" page in my notebook with lot of content on it. See attached.
8. Also shows a "Work" page (Notebook?) that I did not create, empty, created today. See attached.

I do not know if this is related, but my Bookmarks are gone and the Bookmarks Toolbar does not work - in place of what should be the bookmarks are just two giant stretched Left and Right arrow buttons that do nothing.
![zim1](https://user-images.githubusercontent.com/1929777/195725504-b6660dee-a3c4-4e1a-b2ba-d070b06fa196.png)

Edit to say what happened to me sounds similar to to description in #2189
